### ISSUE TITLE
trailing-comma-tuple triggered by split argument list

### ISSUE THREAD

**Issue by BeryJu** [PyCQA/pylint#1424]
### Steps to reproduce
1. Have a line like:
```
    qr_code = forms.CharField(widget=PictureWidget, disabled=True, required=False, \
                             label=_('Scan this Code with your 2FA App.'))
```

### Current behavior

trailing-comma-tuple is detected

### Expected behavior

trailing-comma-tuple should not be detected since it's not a tuple

### pylint --version output

```
pylint 1.7.0,
astroid 1.5.1
Python 3.5.3 (v3.5.3:1880cb95a742, Jan 16 2017, 16:02:32) [MSC v.1900 64 bit (AMD64)]
```

**Comment by BeryJu** [comment]
Better test snippet:
```
"""
pylint test
"""

def test(arg_a, arg_b, arg_c, arg_d, arg_e):
    """
    test
    """
    print(arg_a, arg_b, arg_c, arg_d, arg_e)

test(arg_a='a', arg_b='b', arg_c='c', \
     arg_d='d', arg_e='e')

test('a', 'b', 'c', \
     'd', 'e')
```

Results in:
```
************* Module test
R: 11, 0: Disallow trailing comma tuple (trailing-comma-tuple)

-------------------------------------------------------------------
Your code has been rated at 6.67/10 (previous run: 10.00/10, -3.33)
```

**Comment by aaossa** [comment]
I think that this happens because you are using backslashes (`\\`) where you're not supposed to, because (as `pycodestlye` says) using backslashes between brackets is redundant. If you use `pycodestyle`you should not have this problem. Maybe the message should be different, inviting the user to avoid using a backslash here.
